test/spot-checker.js

```diff
@@ -19,12 +17,30 @@ test('fetchCAR - http', async () => {
     protocol: 'http',
     address: '/dns/frisbii.fly.dev/tcp/443/https',
     cid: KNOWN_CID,
-    stats
+    stats,
+    dagScope: 'all'
   })
-  assertEquals(stats.statusCode, 200, 'stats.statusCode')
-  assertEquals(stats.timeout, false, 'stats.timeout')
-  assertEquals(stats.byteLength, 103, 'stats.byteLength')
-  assertEquals(requests, [`ipfs://${KNOWN_CID}?dag-scope=all&protocols=http&providers=%2Fdns%2Ffrisbii.fly.dev%2Ftcp%2F443%2Fhttps`])
+
+  assertEquals(stats.fullRetrieval.statusCode, 200, 'stats.statusCode')
+  assertEquals(stats.fullRetrieval.timeout, false, 'stats.timeout')
+  assertEquals(stats.fullRetrieval.byteLength, 103, 'stats.byteLength')
```

Comment:
I believe `assertEquals` performs a deep equality check by default, see https://jsr.io/@std/assert@0.226.0/doc/~/assertEquals

```suggestion
  assertEquals(
    stats.fullRetrieval, 
    { statusCode: 200, timeout: false, byteLength: 103 },
    'stats.fullRetrieval'
  )
```